roles/openshift_master_facts/vars/main.yml

```diff
@@ -2,3 +2,15 @@
 openshift_master_config_dir: "{{ openshift.common.config_base }}/master"
 openshift_master_config_file: "{{ openshift_master_config_dir }}/master-config.yaml"
 openshift_master_scheduler_conf: "{{ openshift_master_config_dir }}/scheduler.json"
+# Keep in sync with DefaultAllowedRegistriesForImport in
+# openshift/origin/blob/master/pkg/cmd/server/apis/config/types.go
+default_image_policy_allowed_registries_for_import:
+  allowedRegistriesForImport:
```

Comment:
Could this be maybe generated by the `openshift` binary and fetched from the library module when needed?